packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1266,6 +1363,26 @@ export const CURVE_FANTOM_INFOS: { [name: string]: CurveInfo } = {
         pool: CURVE_FANTOM_POOLS.fUSDT,
         gasSchedule: 587e3,
     }),
+    [CURVE_FANTOM_POOLS.tri_v2]: createCurveExchangePool({
+        tokens: [FANTOM_TOKENS.MIM, FANTOM_TOKENS.fUSDT, FANTOM_TOKENS.USDC],
+        pool: CURVE_FANTOM_POOLS.tri_v2,
+        gasSchedule: 176e3,
+    }),
+    ['aave_exchangeunderlying']: createCurveExchangeUnderlyingPool({
```

Comment:
Any specific reason why this is using the string `aave_exchangeunderlying` instead of pool address as the key?

Reply:
Basically I try to follow the naming tradition for these type of pools. But I will change it to `geist_exchangeunderlying` which seems make more sense